scripts/ci-footprint-and-performance/2-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -xeuo pipefail
+export PS4='\n+ $(date "+%T.%N") ${BASH_SOURCE}:$LINENO \011'
+
+SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+TEST_TIME=$(date +'%Y-%m-%d_%T.%N')
+export TEST_TIME
+
+ARTIFACTS_DIR="$(mktemp -d)"
+export ARTIFACTS_DIR
+
+LOW_LAT_ARTIFACTS="${ARTIFACTS_DIR}/low-latency/${TEST_TIME}"
+export LOW_LAT_ARTIFACTS
```

Comment:
Since these are used elsewhere, can we add comments here to explain a bit what they are?